src/main/java/com/epam/java/training/entity/Product.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.java.training.entity;
+
+import java.io.Serializable;
+import java.util.List;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.epam.java.training.vo.Review;
+@Entity
+@Table(name="product")
+public class Product implements Serializable { 
+	private static final long serialVersionUID = 1L;
+	@Id
+	@GeneratedValue(strategy=GenerationType.AUTO)
+	@Column(name="id")
+    private int id;
+	
+	@Column(name="product")
+    private String product;
```

Comment:
This property can be name otherwise you have to say product.getProduct() to retrieve name